arches/app/media/js/views/components/search/standard-search-view.js

```diff
@@ -0,0 +1,62 @@
+define([
+    'jquery',
+    'underscore',
+    'knockout',
+    'arches',
+    'viewmodels/alert',
+    'views/components/search/base-search-view',
+    'templates/views/components/search/standard-search-view.htm',
+], function($, _, ko, arches, AlertViewModel, BaseSearchViewComponent, standardSearchViewTemplate) {
+    const componentName = 'standard-search-view';
+    const viewModel = BaseSearchViewComponent.extend({ 
+        initialize: function(sharedStateObject) {
+            const self = this;
+            sharedStateObject.componentName = componentName;
+            BaseSearchViewComponent.prototype.initialize.call(this, sharedStateObject);
+            this.defaultQuery = {"paging-filter": "1", "search-view":"standard-search-view", tiles: "true"};
+            
```

Comment:
we could probably remove this line as "required" filters are now defined in the component config
```suggestion
            this.defaultQuery = {"paging-filter": "1", "search-view":"standard-search-view", tiles: "true"};
```